tasks/main.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Create user accounts
+  user:
+    name: "{{ podman_user }}"
+    uid: "{{ podman_user_uid }}"
+
+- name: Configure loginctl to prevent logout causing podman to exit
+  command: loginctl enable-linger {{ podman_user }}
+
+- name: Set podman variable
+  set_fact:
+    podman_conf: podman.conf
+
+- name: Install Podman on RHEL8 server
+  dnf:
+    name:
+      - podman
+    state: latest
+
+- name: Copy podman configuration
+  copy:
+    dest: "/etc/sysctl.d/{{ podman_conf }}"
+    src: "{{ podman_conf }}"
+
+- name: Install machinectl to start podman service
+  dnf:
+    name:
+      - systemd-container
+    state: latest
+
+- name: Enable and start rootless podman.socket
+  command: machinectl shell {{ podman_user }}@.host /bin/sh -c "systemctl --user start podman.socket"
```

Comment:
This only starts the podman backend but not enable it, and it's better to add comment to explain why we need to do it in this way.
```suggestion
# rootless podman backend need to be started with a login session, so
# ansible become methods ("sudo", "su" and etc) doesn't work in this
# case, become method "machinectl" is not flexible enough to fulfill
# the requirement either, we need to use command module as a workaround
- name: Enable and start rootless podman backend
  command: machinectl shell {{ podman_user }}@.host /usr/bin/systemctl --user enable --now podman.socket
```